uniprotkb-rest/src/main/java/org/uniprot/api/uniprotkb/repository/search/impl/UniProtQueryBoostsConfig.java

```diff
@@ -0,0 +1,21 @@
+package org.uniprot.api.uniprotkb.repository.search.impl;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.uniprot.api.common.repository.search.QueryBoosts;
+import org.uniprot.api.common.repository.search.QueryBoostsFileReader;
+
+/**
+ * Created 05/09/19
+ *
+ * @author Edd
+ */
+@Configuration
+public class UniProtQueryBoostsConfig {
+    private static final String BOOSTS_RESOURCE_LOCATION = "uniprotkb-query-boosts.config";
+
```

Comment:
Just following the comment above, here we would add the configurations beans, in the same structure that we have in the configuration file..